content/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter.md

```diff
@@ -0,0 +1,162 @@
+
+```markdown
+docker run --rm --network prometheus \
+    curlimages/curl -s http://my-array-1:8080/metrics
+```
+
+It should produce an output that enumerates the metric keys, value, type and brief help. Example:
+
+```markdown
+# HELP hpenimble_pool_snapshot_used_bytes Total logical size of all snapshots in a storage pool
+# TYPE hpenimble_pool_snapshot_used_bytes gauge
+hpenimble_pool_snapshot_used_bytes{pool="default"} 2.659803136e+09
+```
+
+Next, let's start Prometheus and scrape some metrics.
+
+# Run and Inspect Prometheus
+
+When Prometheus is deployed on Kubernetes, discovery of scrape targets may be configured to be performed dynamically. In this example we need to instruct Prometheus where to find our target. Create a configuration file named `prometheus.yml`:
+
+```yaml
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: "my-array-1"
+    static_configs:
+      - targets: ["my-array-1:8080"]
+```
+
+Create the Prometheus container:
+
+```markdown
+docker run -d --rm --name prometheus-server \
+    --network prometheus \
+    -p 80:9090 \
+    -v $(pwd)/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus:v2.30.3
+```
+
+In a few moments, the Prometheus server should be up and listening on the docker host and accessible from http://your.docker.host.example.com/. 
+
+To inspect if metrics are being collected properly, access the web interface and start typing "hpe" in the query bar.
+
+![Prometheus Query](/img/prometheus-auto.png)
+
+Select a \`volume_reads_per_second\` metric, hit "Execute" and click "Graph". This should produce a filled line graph with all the volumes on the array and their read IOPS. Wait a few minutes if there's no data being presented.
+
+![Prometheus Graph](/img/prometheus-graph.png)
+
+Next, we'll bolt on Grafana and connect to the Prometheus data source.
+
+# Run Grafana and Build Your First Dashboard
+
+In this tutorial we'll just use the built-in dashboards and plugins that come with Grafana. Accessing Grafana is mainly done through the intuitive web UI found on port 3000. Let's start it up:
+
+```markdown
+docker run --rm -d --name grafana \
+    --network prometheus \
+    -p 3000:3000 \
+    grafana/grafana:8.2.2
+```
+
+Next, navigate to the web UI. http://your.docker.host.example.com:3000/, login with user "admin" and password "admin", you'll be prompted to create a new password. Reusing "admin" for this example is perfectly fine.
+
+Next, a data source needs to be added. A panel to do so is available on the main screen.
+
+![Add new data source](/img/grafana-welcome.png)
+
+Next, select "Prometheus".
+
+![Select Prometheus](/img/grafana-select.png)
+
+In the HTTP section of the configuration, enter `http://prometheus-server:9090`. Click "Save & test" at the bottom of the page. A green checkmark should indicate that the "Data source is working".
```

Comment:
Why are you putting quotes around "Data source is working".? Is this something that comes back as a response? If so, it might be nice to include an image of it. But you already have a possibly more important image following - you might just want to remove the quotes.

Reply:
Yes, the quoted string is what's being presented in the UI. I removed the quotes and made it part of the sentence.